nifi-nar-bundles/nifi-kite-bundle/nifi-kite-processors/pom.xml

```diff
@@ -25,8 +25,9 @@
     <packaging>jar</packaging>
 
     <properties>
-        <kite.version>1.0.0</kite.version>
+        <kite.version>1.1.0</kite.version>
         <findbugs-annotations.version>1.3.9-1</findbugs-annotations.version>
+        <jackson.version>2.3.1</jackson.version>
```

Comment:
Any specific reason for adding the jackson dependencies?

Looking at the Kite NAR from master, it appears to already include a newer version of Jackson:
```
ls -l work/nar/extensions/nifi-kite-nar-1.5.0-SNAPSHOT.nar-unpacked/META-INF/bundled-dependencies/ | grep jackson
-rw-r--r--  1 bbende  staff    46968 Oct  4 13:57 jackson-annotations-2.6.0.jar
-rw-r--r--  1 bbende  staff   258833 Oct  4 13:57 jackson-core-2.6.1.jar
-rw-r--r--  1 bbende  staff  1166488 Oct  4 13:57 jackson-databind-2.6.1.jar
-rw-r--r--  1 bbende  staff  1029033 Oct  4 13:57 parquet-jackson-1.4.1.jar
```

Reply:
Well, the thing is... the tests would fail without jackson. So I've looked at the `pom.xml` provided by Kite, and just plugged that into nifi-kite.

Are your tests successful without that addition and kite 1.1.0?

Reply:
Good question :) l didn't get to building it, let me play around with it and get back to you.

Reply:
Ok you were right, here is what is happening...

Kite SDK 1.0.0 has a dependency only on jackson-databind 2.3.1, which transitively depends on jackson-core 2.3.1 and jackson-annotations 2.3.1, but NiFi is overriding all uses of jackson-databind to 2.6.1 so previously you got all three of those jars at 2.6.1.

Kite SDK 1.1.0 directly declares all three jars, so then NiFi overrides jackson-databind to 2.6.1, but jackson-annotations and jackson-core are still at 2.3.1 because that is what Kite declared.

There is another effort going on to clean up some of the dependencies (https://issues.apache.org/jira/browse/NIFI-4297) and I believe part of that is going to remove the part of NiFi that is forcing everything to databind 2.6.1.

I'd prefer to wait for that to happen first, then we can simplify things here and remove all of the jackson stuff from this PR and everything should work. Let me know if there is any issue with that.

Reply:
Sure, I got no problem with that!